stats/lib/app.js

```diff
@@ -31,6 +31,7 @@ export const createApp = ({
       'https://checker-draft.webflow.io',
       'https://www.checker.network',
       'https://leaderboard-f1i.pages.dev',
+      /^https:\/\/[^.]+\.leaderboard-f1i\.pages\.dev$/,
```

Comment:
Can you please add some comments to explain what these additional domains are for?

Reply:
I've added a following comment: "Allow all subdomains of leaderboard-f1i.pages.dev generated by Cloudflare Pages preview deployments". Do you think it needs to be more descriptive?